homeassistant/components/ephember/climate.py

```diff
@@ -100,6 +101,9 @@ def __init__(self, ember, zone):
         if self._hot_water:
             self._attr_supported_features = ClimateEntityFeature.AUX_HEAT
             self._attr_target_temperature_step = None
+        self._attr_supported_features |= (
+            ClimateEntityFeature.TURN_OFF | ClimateEntityFeature.TURN_ON
+        )
```

Comment:
What's the benefit of adding them here instead of changing the original definition for `_attr_supported_features`?

Reply:
Instead of adding them twice into the above I'm just "appending" here to the result of the above statements.
I guess personal preference from my side to write it like this.

Reply:
Ah, missed the "hot water" completely overriding the previously set flags instead of AUX_HEAT being appended there. 👍